src/main/java/dao/TicketDaoImpl.java

```diff
@@ -0,0 +1,92 @@
+package dao;
+
+import model.TicketType;
+import model.ticket.Ticket;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.UUID;
+
+public class TicketDaoImpl implements TicketDao {
+    private final Connection connection;
+
+    public TicketDaoImpl(Connection connection) {
+        this.connection = connection;
+    }
+
+    public boolean save(Ticket ticket) {
+        String sqlCommand = "INSERT INTO public.\"Ticket\" (id, user_id, ticket_type, creation_date)" +
+                " VALUES (?, ?, ?::ticket_type, ?)";
+        try (PreparedStatement statement = connection.prepareStatement(sqlCommand)) {
```

Comment:
Instead of creating Connection in constructor, open it in with try-with-resources construction:
```java
try (Connection connection = ConnectionConfig .getConnection();
      PreparedStatement statement = connection.prepareStatement(sqlCommand)) { ... }
```

Reply:
Refactored connection with try-with-resources construction